Add `osu!taiko World Cup 2025: Quarterfinals Recap` news post (#13117)
ppy/osu-wiki: news/2025/2025-04-10-osutaiko-world-cup-quarterfinals-recap.md

---
layout: post
title: "osu!taiko World Cup 2025: Quarterfinals Recap"
date: 2025-04-10 23:30:00 +0000
---

> Raise your drumsticks and catch up on what went down in the previous week of this years biggest osu!taiko tournament!

Reviewer: ```suggestion
Raise your drumsticks and catch up on what went down in the previous week of this year's biggest osu!taiko tournament!
```

![](/wiki/shared/news/2025-04-10-osutaiko-world-cup-quarterfinals-recap/banner.jpg)

> More intense clashes between ever so determined players fighting to bring the trophy home. Unfortunately, not every team was able to withstand the pressure and we had to say a farewell to some courageous teams. The quarterfinals is where the skill gap usually starts to tighten and by simply giving this a read, you'll find out exactly how this played out last weekend!

Reviewer: ```suggestion
More intense clashes between ever so determined players fighting to bring the trophy home. Unfortunately, not every team was able to withstand the pressure, and we had to say a farewell to some courageous teams. The Quarterfinals are where the skill gap usually starts to tighten and by simply giving this a read, you'll find out exactly how it all played out last weekend!
```

## Mappool showcase

<style>
    .osu-md--news .audio-player {
        max-width: inherit;
    }
</style>

**[Download the mappool here! (100 MB)](https://packs.ppy.sh/P280%20-%20osu%21taiko%20World%20Cup%202025%3A%20Quarterfinals.zip)**

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?autoplay=false&parent=osu.ppy.sh&time=0h43m4s&video=2419648148" allowfullscreen="true" scrolling="no"></iframe>
</div>

![Quarterfinals mappool table](/wiki/shared/news/2025-04-10-osutaiko-world-cup-quarterfinals-recap/mappool.png)

> This weeks [osu! original](/wiki/Community/Bespoke_music) roster has been a blast! Totaling **three** new tracks that spice up this mappool with freshly baked songs from our featured artists. Marmelade butcher, AQUASINE & Supire have worked hard on those following songs to gurantee an exciting-sounding Quarterfinals experience.

Reviewer: ```suggestion
This week's [osu! original](/wiki/Community/Bespoke_music) roster has been a blast! Totalling **three** new tracks that spice up this mappool with freshly baked songs from our featured artists. Marmelade butcher, AQUASINE & Supire have worked hard on those following songs to guarantee an exciting-sounding Quarterfinals experience.
```

| Netherlands ::{ flag=NL }:: | 5 | **6** | ::{ flag=NZ }:: **New Zealand** |
| Australia ::{ flag=AU }:: | 5 | **6** | ::{ flag=FR }:: **France** |
| **South Korea** ::{ flag=KR }:: | **6** | 1 | ::{ flag=CN }:: China |
| Vietnam ::{ flag=VN }:: | 4 | **6** | ::{ flag=SK }:: **Slovakia** |
| **United Kingdom** ::{ flag=GB }:: | **6** | 0 | ::{ flag=HK }:: Hong Kong |
| Singapore ::{ flag=SG }:: | 5 | **6** | ::{ flag=TW }:: **Taiwan** |
| **Russian Federation** ::{ flag=RU }:: | **6** | 3 | ::{ flag=PH }:: Philippines |
| **Indonesia** ::{ flag=ID }:: | **6** | 2 | ::{ flag=CA }:: Canada |
| **Türkiye** ::{ flag=TR }:: | **6** | -1 | ::{ flag=PL }:: Poland |
| **Finland** ::{ flag=FI }:: | **6** | 0 | ::{ flag=CH }:: Switzerland |
| United States ::{ flag=US }:: | 5 | **6** | ::{ flag=CL }:: **Chile** |
| **France** ::{ flag=FR }:: | **6** | 1 | ::{ flag=NZ }:: New Zealand |
| **Japan** ::{ flag=JP }:: | **6** | 0 | ::{ flag=MY }:: Malaysia |
| **Türkiye** ::{ flag=TR }:: | **6** | 1 | ::{ flag=ID }:: Indonesia |
| **Russian Federation** ::{ flag=RU }:: | **6** | 1 | ::{ flag=FI }:: Finland |
| **United Kingdom** ::{ flag=GB }:: | **6** | 0 | ::{ flag=SK }:: Slovakia |

> Switching things up to a Best of 13 format, the Quarterfinals had some very close matches for some teams. Interestingly enough: Every time the tiebreaker was played, the lower seeded team came out as the victor. Not all of them were able to pull through and come out on top however, so the following teams were unfortunately eliminated in the Quarterfinals: ::{ flag=PH }:: Philippines, ::{ flag=CH }:: Switzerland, ::{ flag=HK }:: Hong Kong, ::{ flag=VN }:: Vietnam, ::{ flag=PL }:: Poland, ::{ flag=CA }:: Canada, ::{ flag=AU }:: Australia, ::{ flag=NL }:: Netherlands, ::{ flag=FI }:: Finland, ::{ flag=SK }:: Slovakia, ::{ flag=ID }:: Indonesia & ::{ flag=NZ }:: New Zealand.

Reviewer: ```suggestion
Switching things up to a Best of 13 format, the Quarterfinals had some very close matches for some teams. Interestingly enough: every time the tiebreaker was played, the lower seeded team came out as the victor. Not all of them were able to pull through and come out on top however, so the following teams were unfortunately eliminated: ::{ flag=PH }:: Philippines, ::{ flag=CH }:: Switzerland, ::{ flag=HK }:: Hong Kong, ::{ flag=VN }:: Vietnam, ::{ flag=PL }:: Poland, ::{ flag=CA }:: Canada, ::{ flag=AU }:: Australia, ::{ flag=NL }:: Netherlands, ::{ flag=FI }:: Finland, ::{ flag=SK }:: Slovakia, ::{ flag=ID }:: Indonesia & ::{ flag=NZ }:: New Zealand.
```

> Check out detailed statistics for this round [here](https://docs.google.com/spreadsheets/d/1HYP2llOprad492cK_JaKEV9B79wMk1OYN8c8xSpa2xI).

Reviewer 2: more flair

```suggestion
As always, the intricate details for this round can be found in [one of our statistics spreadsheets](https://docs.google.com/spreadsheets/d/1HYP2llOprad492cK_JaKEV9B79wMk1OYN8c8xSpa2xI), as prepared by [our head world cup statistician](https://osu.ppy.sh/users/10000899).
```

## Match highlights

### ::{ flag=AU }:: Australia vs. ::{ flag=FR }:: France

**[View the match information here!](https://osu.ppy.sh/community/matches/117698638)**

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?autoplay=false&parent=osu.ppy.sh&time=0h4m42&video=2424570850" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

> Wrapping up this train of tiebreaker upsets, we have perhaps the most surprising of the week: Team Australia (#8) vs. Team France (#19)—not only the largest difference in seeding, but with Team France holding a total of zero pick'em votes, truly nobody saw this coming.

Reviewer: ```suggestion
Wrapping up this train of tiebreaker upsets, we have perhaps the most surprising of the week: Team Australia (#8) vs. Team France (#19) — not only the largest difference in seeding, but with Team France holding a total of zero pick'em votes, truly nobody saw this coming.
```

> Australia began by playing it safe with [Marmalade Butcher - =/=Virtual Futural](https://osu.ppy.sh/beatmapsets/2347292#taiko/5048649) (NM1), taking their first point while testing the waters. However, France quickly showed that Australia might have gone fishing in the wrong lake, with a hundred-thousand-point win on [Yunosuke - Ziqqurat (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2347287#taiko/5048627) (NM6).

Reviewer: ```suggestion
Australia began by playing it safe with [Marmalade Butcher - =/=Virtual Futural](https://osu.ppy.sh/beatmapsets/2347292#taiko/5048649) (NM1), taking their first point while testing the waters. However, France quickly showed that Australia might have gone fishing in the wrong lake, with a 100,000 point win on [Yunosuke - Ziqqurat (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2347287#taiko/5048627) (NM6).
```

> Having stirred the pot, the two teams began trading points: Australia winning [Mono. - skip&step](https://osu.ppy.sh/beatmapsets/2069452#taiko/4330009) (HR1), France taking [Hoshimachi Suisei - BIBBIDIBA(AL-1S & UZQueen's Edit)](https://osu.ppy.sh/beatmapsets/2207301#taiko/4673960) (NM2), followed by an Australian accuracy advantage on [DARK_GENE - GREEN MAMMALS](https://osu.ppy.sh/beatmapsets/2347276#taiko/5048593) (FM1), and an extremely narrow five-thousand-point French victory on [Pmarusama. - Tokimeki Broker](https://osu.ppy.sh/beatmapsets/2189313#taiko/4630501) (DT2).

Reviewer 2: still in the thousands so the spelled-out emphasis only makes it slightly harder to parse the numeral at a glance

```suggestion
Having stirred the pot, the two teams began trading points: Australia winning [Mono. - skip&step](https://osu.ppy.sh/beatmapsets/2069452#taiko/4330009) (HR1), France taking [Hoshimachi Suisei - BIBBIDIBA(AL-1S & UZQueen's Edit)](https://osu.ppy.sh/beatmapsets/2207301#taiko/4673960) (NM2), followed by an Australian accuracy advantage on [DARK_GENE - GREEN MAMMALS](https://osu.ppy.sh/beatmapsets/2347276#taiko/5048593) (FM1), and an extremely narrow 5,000-point French victory on [Pmarusama. - Tokimeki Broker](https://osu.ppy.sh/beatmapsets/2189313#taiko/4630501) (DT2).
```

> While they remained tied at this point, Australia proved their dominance in reading skill with back-to-back wins on [yuha - Nou#m7 (feat. KAFU)](https://osu.ppy.sh/beatmapsets/2190945#taiko/4634009) (HD2) and [AQUASINE - you wouldn't download a bird.](https://osu.ppy.sh/beatmapsets/2347271#taiko/5048585) (NM4)... but France did the same on the more finicky [Caravan Palace - City Cook](https://osu.ppy.sh/beatmapsets/2147535#taiko/4523594) (DT1) and [Schwank - BPM](https://osu.ppy.sh/beatmapsets/2347031#taiko/5047553) (FM2)! At this point, the counting connoisseurs among the viewers knew it was time for yet another tiebreaker.

Reviewer: ```suggestion
While they remained tied at this point, Australia proved their dominance in reading skill with back-to-back wins on [yuha - Nou#m7 (feat. KAFU)](https://osu.ppy.sh/beatmapsets/2190945#taiko/4634009) (HD2) and [AQUASINE - you wouldn't download a bird.](https://osu.ppy.sh/beatmapsets/2347271#taiko/5048585) (NM4)... France would do the same on the more finicky [Caravan Palace - City Cook](https://osu.ppy.sh/beatmapsets/2147535#taiko/4523594) (DT1) and [Schwank - BPM](https://osu.ppy.sh/beatmapsets/2347031#taiko/5047553) (FM2)! At this point, the counting connoisseurs among the viewers knew it was time for yet another tiebreaker.
```

> [Supire - Dead Aire](https://osu.ppy.sh/beatmapsets/2347284#taiko/5048624) (TB) rolled in to raise the tension, and despite an incredible Hidden performance from ::{ flag=AU }:: [Jaye](https://osu.ppy.sh/users/4841352), Team France proved to be the bigger fish. Three sets of scores over 900,000 bringing them a well-deserved victory.

Reviewer 2: missing predicate in the sentence

```suggestion
[Supire - Dead Aire](https://osu.ppy.sh/beatmapsets/2347284#taiko/5048624) (TB) rolled in to raise the tension, and despite an incredible Hidden performance from ::{ flag=AU }:: [Jaye](https://osu.ppy.sh/users/4841352), Team France proved to be the bigger fish. Three sets of scores over 900,000 brought them a well-deserved victory.
```

> [Supire - Dead Aire](https://osu.ppy.sh/beatmapsets/2347284#taiko/5048624) (TB) rolled in to raise the tension, and despite an incredible Hidden performance from ::{ flag=AU }:: [Jaye](https://osu.ppy.sh/users/4841352), Team France proved to be the bigger fish. Three sets of scores over nine-hundred-thousand bringing them a well-deserved victory.

Reviewer: ```suggestion
[Supire - Dead Aire](https://osu.ppy.sh/beatmapsets/2347284#taiko/5048624) (TB) rolled in to raise the tension, and despite an incredible Hidden performance from ::{ flag=AU }:: [Jaye](https://osu.ppy.sh/users/4841352), Team France proved to be the bigger fish. Three sets of scores over 900,000 bringing them a well-deserved victory.
```

### ::{ flag=SG }:: Singapore vs. ::{ flag=TW }:: Taiwan

**[View the match information here!](https://osu.ppy.sh/community/matches/117700529)**

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?autoplay=false&parent=osu.ppy.sh&time=1h6m16s&video=2424643241" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

One of the most intense matches so far, Singapore vs. Taiwan has surely kept viewers on their toes all the way through.

> Singapore held some solid momentum at the very start through [Yunosuke - Ziqqurat (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2347287#taiko/5048627) (NM6) and the majority of [yuha - Nou#m7 (feat. KAFU)](https://osu.ppy.sh/beatmapsets/2190945#taiko/4634009) (HD2). However, with one slip-up in the ending section of HD2, Taiwan jumped on the opportunity instantly and chipped away the score difference slowly but steadily, until it came to the **last second of the map**. Yes, you heard that right, Taiwan managed to clutch up the lead at exactly the last second, cementing the fact that they won't be going down easy in this match.

Reviewer 2: ```suggestion
Singapore held some solid momentum at the very start through [Yunosuke - Ziqqurat (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2347287#taiko/5048627) (NM6) and the majority of [yuha - Nou#m7 (feat. KAFU)](https://osu.ppy.sh/beatmapsets/2190945#taiko/4634009) (HD2). However, with one slip-up in the ending section of HD2, Taiwan jumped on the opportunity instantly and chipped away the score difference slowly but steadily, until it came to the **last second of the map**. Yes, you heard that right, Taiwan managed to clutch up the lead at exactly the last second, cementing the fact that they won't be going down easily in this match.
```

> After a very intense accuracy battle and 6 FCs on [Mono. - skip&step](https://osu.ppy.sh/beatmapsets/2069452#taiko/4330009) (HR1) players headed onto the next pick, [DARK_GENE - GREEN MAMMALS (hyper mix)](https://osu.ppy.sh/beatmapsets/2347276#taiko/5048593) (FM1) where all 6 players decided that it's time for a full HDHR lobby. Despite the uphill battle against Team Taiwan's combo, Team Singapore was able to secure the point just by hitting the notes more accurately.

Reviewer 2: ```suggestion
After a very intense accuracy battle and 6 FCs on [Mono. - skip&step](https://osu.ppy.sh/beatmapsets/2069452#taiko/4330009) (HR1), players headed into the next pick, [DARK_GENE - GREEN MAMMALS (hyper mix)](https://osu.ppy.sh/beatmapsets/2347276#taiko/5048593) (FM1), where all 6 players decided that it's time for a full HDHR lobby. Despite the uphill battle against Team Taiwan's combo, Team Singapore was able to secure the point just by hitting the notes more accurately.
```

> After a very intense accuracy battle and 6 FCs on [Mono. - skip&step](https://osu.ppy.sh/beatmapsets/2069452#taiko/4330009) (HR1) players headed onto the next pick, [DARK_GENE - GREEN MAMMALS (hyper mix)](https://osu.ppy.sh/beatmapsets/2347276#taiko/5048593) (FM1) where all 6 players decided that it's time for a full HDHR lobby. Despite the uphill battle against Team Taiwan's combo, Team Singapore was able to secure the point just by hitting the notes more accurately.

Reviewer: ```suggestion
After a very intense accuracy battle and 6 FCs on [Mono. - skip&step](https://osu.ppy.sh/beatmapsets/2069452#taiko/4330009) (HR1) players headed onto the next pick, [DARK_GENE - GREEN MAMMALS (hyper mix)](https://osu.ppy.sh/beatmapsets/2347276#taiko/5048593) (FM1) where all 6 players decided that it's time for a full HDHR lobby. Despite the uphill battle against team Taiwan's combo, Team Singapore was able to secure the point just by hitting the notes more accurately.
```

> Only a couple of picks and intense neck in neck battles later, the unbelievable happens. Once again, Team Taiwan won [Pmarusama. - Tokimeki Broker](https://osu.ppy.sh/beatmapsets/2189313#taiko/4630501) after ::{ flag=SG }:: [Prehistoria](https://osu.ppy.sh/users/8364237) found an unfortunate miss on one of the last patterns in the map. Score lead quickly jumped over to the side of the Taiwanese team within the last few seconds of the map and brought another very fierce match to tiebreaker.

Reviewer 2: ```suggestion
Only a couple of picks and intense neck-and-neck battles later, the unbelievable happened. Once again, Team Taiwan won [Pmarusama. - Tokimeki Broker](https://osu.ppy.sh/beatmapsets/2189313#taiko/4630501) after ::{ flag=SG }:: [Prehistoria](https://osu.ppy.sh/users/8364237) found an unfortunate miss on one of the last patterns in the map. The score lead quickly jumped over to the side of the Taiwanese team within the last few seconds of the map and brought another very fierce match to the tiebreaker.
```

> Only a couple of picks and intense neck in neck battles later, the unbelievable happens. Once again, Team Taiwan won [Pmarusama. - Tokimeki Broker](https://osu.ppy.sh/beatmapsets/2189313#taiko/4630501) after ::{ flag=SG }:: [Prehistoria](https://osu.ppy.sh/users/8364237) found an unfortunate miss on one of the last patterns in the map. Score lead quickly jumped over to the side of the Taiwanese team within the last few seconds of the map and brought another very fierce match to tiebreaker.

Reviewer: ```suggestion
Only a couple of picks and intense neck in neck battles later, the unbelievable happens. Once again, Team Taiwan won [Pmarusama. - Tokimeki Broker](https://osu.ppy.sh/beatmapsets/2189313#taiko/4630501) after ::{ flag=SG }:: [Prehistoria](https://osu.ppy.sh/users/8364237) found an unfortunate miss on one of the last patterns in the map. Score lead quickly jumped over to the side of the Taiwanese team within the last few seconds of the map and brought another very fierce match to tiebreaker.
```

> After both teams used their well deserved 2-minute timeout to take a breather, everything came down to one last map. With [Supire - Dead Aire](https://osu.ppy.sh/beatmapsets/2347284#taiko/5048624) (TB) already playing in the background, the players knew they only had one last chance to come out on top. One player, namely ::{ flag=TW }:: [HarryAda](https://osu.ppy.sh/users/12892087) was brave enough to go into this tiebreaker with the Hidden mod enabled. This kept the score lead ever so slightly on the side of the Taiwanese team. Despite ::{ flag=SG }:: [Blerargh](https://osu.ppy.sh/users/7609510)'s high accuracy of 99.57%, Team Taiwan was able to pull through with the previously mentioned Hidden score multiplier, supported by ::{ flag=TW }:: [HiGreeks](https://osu.ppy.sh/users/16536516)' combo of 2,349. It was just not enough for Team Singapore.

Reviewer 2: the score multiplier wasn't actually aforementioned

```suggestion
After both teams used their well-deserved 2-minute timeout to take a breather, everything came down to one last map. With [Supire - Dead Aire](https://osu.ppy.sh/beatmapsets/2347284#taiko/5048624) (TB) already playing in the background, the players knew they only had one last chance to come out on top. One player, namely ::{ flag=TW }:: [HarryAda](https://osu.ppy.sh/users/12892087), was brave enough to go into this tiebreaker with the Hidden mod enabled. This kept the score lead ever so slightly on the side of the Taiwanese team. Despite ::{ flag=SG }:: [Blerargh](https://osu.ppy.sh/users/7609510)'s high accuracy of 99.57%, Team Taiwan was able to pull through with the Hidden mod's score multiplier, supported by ::{ flag=TW }:: [HiGreeks](https://osu.ppy.sh/users/16536516)' combo of 2,349. It was just not enough for Team Singapore.
```

### ::{ flag=US }:: United States vs. ::{ flag=CL }:: Chile

**[View the match information here!](https://osu.ppy.sh/community/matches/117707419)**

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?autoplay=false&parent=osu.ppy.sh&time=0h4m40s&video=2425207564" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

> One of the most anticipated matches of the entire weekend was none other than the winners bracket clash between Team United States (#3) and Team Chile (#6). With 88.5% of the pick'em vote, viewers were convinced that Team United States had the upper hand in this close match up, perhaps not expecting that another tiebreaker was on its way!

Reviewer 2: ```suggestion
One of the most anticipated matches of the entire weekend was none other than the winners' bracket clash between Team United States (#3) and Team Chile (#6). With 88.5% of the pick'em vote, viewers were convinced that Team United States had the upper hand in this close matchup, perhaps not expecting that another tiebreaker was on its way!
```

> Everything started smoothly with Team United States experimenting on the new slot of the pool, picking [Yunosuke - Ziqqurat (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2347287#taiko/5048627) (NM6), showcasing a strong three way full-combo and convincing win. Chile responded with [Schwank - BPM](https://osu.ppy.sh/beatmapsets/2347031#taiko/5047553) (FM2) on the opposite side of the pool, displaying their mod prowess with a victory on the triple HDHR.

Reviewer: ```suggestion
Everything started smoothly with Team United States experimenting on the new slot of the pool, picking [Yunosuke - Ziqqurat (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2347287#taiko/5048627) (NM6), showcasing a strong three-way full-combo and convincing win. Chile responded with [Schwank - BPM](https://osu.ppy.sh/beatmapsets/2347031#taiko/5047553) (FM2) on the opposite side of the pool, displaying their mod prowess with a victory on the triple HDHR.
```

> This trend was short-lived as Team Chile won a breakpoint on [Hoshimachi Suisei - BIBBIDIBA(AL-1S & UZQUEEN's Edit)](https://osu.ppy.sh/beatmapsets/2207301#taiko/4673960) (NM2), with Team United States immediately winning it back on [yuha - Nou#m7 (feat. KAFU)](https://osu.ppy.sh/beatmapsets/2190945#taiko/4634009) (HD2). However, Team Chile managed to keep the momentum in their favour as they took another breakpoint on [Umeboshi Chazuke - Lollipop*driveR](https://osu.ppy.sh/beatmapsets/2347262#taiko/5048575) (NM3) and winning their own pick of [reallerr - a funk that never stops](https://osu.ppy.sh/beatmapsets/2347270#taiko/5048584) (HD1).

Reviewer 2: ```suggestion
This trend was short-lived as Team Chile won a break point on [Hoshimachi Suisei - BIBBIDIBA(AL-1S & UZQUEEN's Edit)](https://osu.ppy.sh/beatmapsets/2207301#taiko/4673960) (NM2), with Team United States immediately winning it back on [yuha - Nou#m7 (feat. KAFU)](https://osu.ppy.sh/beatmapsets/2190945#taiko/4634009) (HD2). However, Team Chile managed to keep the momentum in their favour as they took another break point on [Umeboshi Chazuke - Lollipop*driveR](https://osu.ppy.sh/beatmapsets/2347262#taiko/5048575) (NM3) and won their own pick of [reallerr - a funk that never stops](https://osu.ppy.sh/beatmapsets/2347270#taiko/5048584) (HD1).
```

> This trend was short lived as Team Chile won a breakpoint on [Hoshimachi Suisei - BIBBIDIBA(AL-1S & UZQUEEN's Edit)](https://osu.ppy.sh/beatmapsets/2207301#taiko/4673960) (NM2) with Team United States immediately winning it back on [yuha - Nou#m7 (feat. KAFU)](https://osu.ppy.sh/beatmapsets/2190945#taiko/4634009) (HD2). However, Team Chile managed to keep the momentum in their favour as they took another breakpoint on [Umeboshi Chazuke - Lollipop*driveR](https://osu.ppy.sh/beatmapsets/2347262#taiko/5048575) (NM3) and winning their own pick of [reallerr - a funk that never stops](https://osu.ppy.sh/beatmapsets/2347270#taiko/5048584) (HD1).

Reviewer: ```suggestion
This trend was short-lived as Team Chile won a breakpoint on [Hoshimachi Suisei - BIBBIDIBA(AL-1S & UZQUEEN's Edit)](https://osu.ppy.sh/beatmapsets/2207301#taiko/4673960) (NM2), with Team United States immediately winning it back on [yuha - Nou#m7 (feat. KAFU)](https://osu.ppy.sh/beatmapsets/2190945#taiko/4634009) (HD2). However, team Chile managed to keep the momentum in their favour as they took another breakpoint on [Umeboshi Chazuke - Lollipop*driveR](https://osu.ppy.sh/beatmapsets/2347262#taiko/5048575) (NM3) and winning their own pick of [reallerr - a funk that never stops](https://osu.ppy.sh/beatmapsets/2347270#taiko/5048584) (HD1).
```

Team United States did not let this ruin their morale as they output three incredible performances in a row on [CBR - Love Maker!!](https://osu.ppy.sh/beatmapsets/2347267#taiko/5048581) (NM5), Team Chile's [AQUASINE - you wouldn't download a bird.](https://osu.ppy.sh/beatmapsets/2347271#taiko/5048585) (NM4) pick, and [Caravan Palace - City Cook](https://osu.ppy.sh/beatmapsets/2147535#taiko/4523594) (DT1), bringing them to the highly crucial match point. With only a few options left, Team Chile took their chances on [+TEK - starlove](https://osu.ppy.sh/beatmapsets/2213428#taiko/4689756) (HR2) and denied Team United States the easy way out with a fifth point of their own, forcing the match into the tiebreaker!

> Once again, [Supire - Dead Aire](https://osu.ppy.sh/beatmapsets/2347284#taiko/5048624) (TB) descended onto the stage where two players on each team took their chances by playing with Hidden. Team Chile managed to secure an early lead and never let it go, narrowly securing their place in the Semifinals winners bracket and upsetting Team United States by 36,748 points.

Reviewer 2: "took their chances" was just written last paragraph

```suggestion
Once again, [Supire - Dead Aire](https://osu.ppy.sh/beatmapsets/2347284#taiko/5048624) (TB) descended onto the stage, where two players on each team looked to seize the advantage by playing with Hidden. Team Chile ended up managing to secure an early lead and never let it go, narrowly securing their place in the Semifinals winners' bracket and upsetting Team United States by 36,748 points.
```

## Score highlights

> With a few really good scores this last weekend, a couple of them definitely stood out from the rest, which we by no means want to withhold from you!

Reviewer: ```suggestion
A couple of scores from last weekend stood out from the rest, which we by no means want to withhold from you!
```

> - ::{ flag=CL }:: [ulko](https://osu.ppy.sh/users/1263669)'s stunning sliver SS on [yuha - Nou#m7 (feat. KAFU)](https://osu.ppy.sh/beatmapsets/2190945#taiko/4634009) (HD2) in their match against the United States, despite not winning the map, makes this one of the scariest threats to Team South Korea, as they will be facing Chile in this weekend's Semifinals.

Reviewer 2: ```suggestion
- ::{ flag=CL }:: [ulko](https://osu.ppy.sh/users/1263669)'s stunning silver SS on [yuha - Nou#m7 (feat. KAFU)](https://osu.ppy.sh/beatmapsets/2190945#taiko/4634009) (HD2) in their match against the United States, despite not winning the map, makes this one of the scariest threats to Team South Korea, as they will be facing Chile in this weekend's Semifinals.
```

> - ::{ flag=CL }:: ulko's stunning sliver SS on [yuha - Nou#m7 (feat. KAFU)](https://osu.ppy.sh/beatmapsets/2190945#taiko/4634009) (HD2) in their match against the United States, despite not winning the map, makes this one of the scariest threats to Team South Korea, as they will be facing Chile in this weekends Semifinals.

Reviewer: ```suggestion
- ::{ flag=CL }:: [ulko](https://osu.ppy.sh/users/1263669)'s stunning sliver SS on [yuha - Nou#m7 (feat. KAFU)](https://osu.ppy.sh/beatmapsets/2190945#taiko/4634009) (HD2) in their match against the United States, despite not winning the map, makes this one of the scariest threats to Team South Korea, as they will be facing Chile in this weekend's Semifinals.
```

> - Not one, not two, but **three** players have shown, that even 39-second long streams is nothing they are afraid of. After 3:20 minutes of intense stream baking, ::{ flag=KR }:: Peaceful, ::{ flag=KR }:: Rosemarie and ::{ flag=US }:: SilverTyrant have hit all the notes with perfect accuracy on [CBR - Love Maker!!](https://osu.ppy.sh/beatmapsets/2347267#taiko/5048581) (NM5).

Reviewer: ```suggestion
- Not one, not two, but **three** players have shown, that even 39-second long streams is nothing they are afraid of. After 3:20 minutes of intense stream baking, ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027), ::{ flag=KR }:: [Rosemarie](https://osu.ppy.sh/users/2228401) and ::{ flag=US }:: [SilverTyrant](https://osu.ppy.sh/users/14356353) have hit all the notes with perfect accuracy on [CBR - Love Maker!!](https://osu.ppy.sh/beatmapsets/2347267#taiko/5048581) (NM5).
```

> - Not one, not two, but **three** players have shown, that even 39-second long streams is nothing they are afraid of. After 3:20 minutes of intense stream baking, ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027), ::{ flag=KR }:: [Rosemarie](https://osu.ppy.sh/users/2228401) and ::{ flag=US }:: [SilverTyrant](https://osu.ppy.sh/users/14356353) have hit all the notes with perfect accuracy on [CBR - Love Maker!!](https://osu.ppy.sh/beatmapsets/2347267#taiko/5048581) (NM5).

Reviewer 2: sounds slightly off

```suggestion
- Not one, not two, but **three** players have shown, that even 39-second-long streams are nothing to be afraid of (for them at least). After 3:20 minutes of intense stream baking, ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027), ::{ flag=KR }:: [Rosemarie](https://osu.ppy.sh/users/2228401) and ::{ flag=US }:: [SilverTyrant](https://osu.ppy.sh/users/14356353) have hit all the notes with perfect accuracy on [CBR - Love Maker!!](https://osu.ppy.sh/beatmapsets/2347267#taiko/5048581) (NM5).
```

---

> We will be back for more next week but before that you should keep a close watch on [osulive Twitch channel](https://www.twitch.tv/osulive) to make sure you don't miss out on the next couple of exciting matches this upcoming semifinals weekend. Additionally, check out the schedules on the [TWC 2025 wiki page](/wiki/Tournaments/TWC/2025) and plan your weekend accordingly to not miss any of the action.

Reviewer: ```suggestion
We will be back for more next week, but before that you should keep a close watch on [osulive Twitch channel](https://www.twitch.tv/osulive) to make sure you don't miss out on the next couple of exciting matches this upcoming Semifinals weekend. Additionally, check out the schedules on the [TWC 2025 wiki page](/wiki/Tournaments/TWC/2025) and plan your weekend accordingly to not miss any of the action.
```

> We will be back for more next week, but before that you should keep a close watch on [osulive Twitch channel](https://www.twitch.tv/osulive) to make sure you don't miss out on the next couple of exciting matches this upcoming Semifinals weekend. Additionally, check out the schedules on the [TWC 2025 wiki page](/wiki/Tournaments/TWC/2025) and plan your weekend accordingly to not miss any of the action.

Reviewer 2: ```suggestion
We will be back for more next week, but before that you should keep a close watch on the [osulive Twitch channel](https://www.twitch.tv/osulive) to make sure you don't miss out on the next couple of exciting matches this upcoming Semifinals weekend. Additionally, check out the schedules on the [TWC 2025 wiki page](/wiki/Tournaments/TWC/2025) and plan your weekend accordingly to not miss any of the action — starting with China (#7) vs. France (#19) at 12:00 UTC this Saturday!
```
